protoutil/txutils.go

```diff
@@ -14,9 +14,12 @@ import (
 	"github.com/golang/protobuf/proto"
 	"github.com/hyperledger/fabric-protos-go/common"
 	"github.com/hyperledger/fabric-protos-go/peer"
+	"github.com/hyperledger/fabric/common/flogging"
 	"github.com/pkg/errors"
 )
 
+var logger = flogging.MustGetLogger("protoutil")
```

Comment:
The logger of the caller may be decorated with all kinds of useful information such as channel name, etc. 

Let's inject a logger into the method call instead of using a static one here.